features/environment.py

```diff
@@ -344,6 +345,12 @@ def backup(self, dest=os.path.join('data', 'basebackup')):
                         '--datadir=' + os.path.join(self._work_directory, dest),
                         '--dbname=' + self.backup_source])
 
+    def read_patroni_log(self, type):
+        try:
+            with open(str(os.path.join(self._output_dir or '', self._name + ".log"))) as f:
+                return [line for line in f.readlines() if line[24:24 + len(type)] == type]
```

Comment:
It is actually log `level`, not `type`. Also, codacy [complains](https://app.codacy.com/gh/zalando/patroni/pullRequest?prid=12071813) about it because `type` is a built-in and it is better to avoid using it as a variable name.
```suggestion
    def read_patroni_log(self, level):
        try:
            with open(str(os.path.join(self._output_dir or '', self._name + ".log"))) as f:
                return [line for line in f.readlines() if line[24:24 + len(level)] == level]
```